app/models/katello/glue/pulp/node.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  module Glue::Pulp::Node
```

Comment:
yeah a lot of this overlaps with the node removal work that is ongoing that will add cert auth to the capsules.  Would you want to meet to discuss this more?